_includes/embed/bilibili.html

```diff
@@ -0,0 +1,7 @@
+<iframe
+  class="embed-video bilibili"
+  loading="lazy"
+  src="https://player.bilibili.com/player.html?bvid={{ include.id }}&danmaku=1"
+  frameborder="0"
+  allowfullscreen="true"
+></iframe>
```

Comment:
I got the `<iframe>` code snippet from the [bilibili.com](https://bilibili.com) and thought it would be better defined this way.